File: examples/time/zone/node/getTheTimezoneInfoForASpecificLocation.js
const { TimeService } = require("m3o/time");

// Get the timezone info for a specific location
async function getTheTimezoneInfoForAspecificLocation() {
  let timeService = new TimeService(process.env.MICRO_API_TOKEN);
  let rsp = await timeService.zone({
    location: "London",
  });
  console.log(rsp);
}

getTheTimezoneInfoForAspecificLocation();

File: examples/time/zone/go/getTheTimezoneInfoForASpecificLocation.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/time"
)

// Get the timezone info for a specific location
func GetTheTimezoneInfoForAspecificLocation() {
	timeService := time.NewTimeService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := timeService.Zone(&time.ZoneRequest{
		Location: "London",
	})
	fmt.Println(rsp, err)
}
